libraries/ui/src/main/java/androidx/media3/ui/SubtitlePainter.java

```diff
@@ -95,9 +99,11 @@
   private int textTop;
   private int textPaddingX;
   private @MonotonicNonNull Rect bitmapRect;
+  private final int horizontalPadding;
+  private final List<PaddingLineBackgroundSpan.PaddingSpanInfo> paddingSpanInfos = new ArrayList<>();
```

Comment:
Do we need the `paddingSpanInfos` defined here? Can't we create a new one in `setupPaddingSpan(..)`?

Reply:
resolved.